src/main/java/com/TableTalk/Enterprise/dao/IGameDAO.java

```diff
@@ -7,9 +7,35 @@
 import java.util.List;
 
 public interface IGameDAO {
+    /**
+     * Generates a game
+     *
+     * @param game
+     * @return
+     * @throws Exception
+     */
     Game save (Game game) throws Exception;
+
+    /**
+     * @return
```

Comment:
JavaDoc is only valuable if it describes a method, over and above what already exists there.  This JavaDoc is not adding value.